echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/orca/PipelineInitiator.java

```diff
@@ -115,17 +124,23 @@ public void startPipeline(Pipeline pipeline) {
             // itself
             boolean propagateAuth =
                 pipeline.getTrigger() != null && pipeline.getTrigger().isPropagateAuth();
-            log.debug("Planning templated pipeline {} before triggering", pipeline.getId());
+            log.debug("Planning templated pipeline {} before triggering", pipeline);
             pipeline = pipeline.withPlan(true);
 
             try {
+              Map pipelineToPlan = objectMapper.convertValue(pipeline, Map.class);
               Map resolvedPipelineMap =
-                  orca.plan(objectMapper.convertValue(pipeline, Map.class), true);
+                  AuthenticatedRequest.allowAnonymous(() -> orca.plan(pipelineToPlan, true));
               pipeline = objectMapper.convertValue(resolvedPipelineMap, Pipeline.class);
             } catch (RetrofitError e) {
-              log.warn(
-                  "Pipeline planning failed: \n{}",
-                  new String(((TypedByteArray) e.getResponse().getBody()).getBytes()));
+              String orcaResponse = "N/A";
+
+              if (e.getResponse() != null && e.getResponse().getBody() != null) {
+                new String(((TypedByteArray) e.getResponse().getBody()).getBytes());
```

Comment:
should this be assigned to something? like `orcaResponse`?

Reply:
um... yes. thank you! i refactored this and messed up